src/mono/wasm/debugger/tests/debugger-test/debugger-evaluate-test.cs

```diff
@@ -417,39 +417,33 @@ public string CallMethodWithChar(char parm)
             }
         }
 
-        public static void EvaluateMethods()
+        public static async Task EvaluateMethods()
```

Comment:
```suggestion
        public static async Task EvaluateMethodsAsync()
```

This will make it obvious that we are dealing with an async method, and should help in reading the tests.